### ISSUE TITLE
Some thoughts on the README

### ISSUE THREAD

**Issue by c089** [rendrjs/rendr-app-template#36]
While the README generally did a good job on getting us started on exploring the app-template, there were a few things we were confused by while reading the documentation:

> We set the property identifier on the view constructor to aid in the view hydration process. More on that later.

However, there is no mention of "hydration" further down in the README... We are still wondering why it is required to have this property on View constructors, but not e.g. on the Controller constructor.

We also think the following statement

> Because of a trick in the way we do Stitch packaging, this module path works in the browser as well.

justifies more explanation, at least a pointer to where that "trick" is implemented so we can understand what's going on.

Please let us know if this it the right place to place to ask this kind of question. We'd also like to aid in the documentation process so I'll keep opening pull requests even for typos or when we try to make things clearer while we explore and evaluate rendr for our needs - unless you don't like that kind of micro-improvement :)

Thanks for the great project!

**Comment by spikebrehm** [comment]
Agreed Christoph, thanks for bringing this up.  I'll revamp the docs with
the upcoming 0.5.0 release.
